framework_tvb/tvb/tests/framework/adapters/analyzers/adapters_test.py

```diff
@@ -48,14 +48,13 @@
 from tvb.adapters.datatypes.h5.temporal_correlations_h5 import CrossCorrelationH5
 from tvb.adapters.datatypes.h5.time_series_h5 import TimeSeriesRegionH5
 from tvb.core.neocom import h5
-from tvb.tests.framework.adapters.analyzers.fft_test import make_ts_from_op
 from tvb.tests.framework.core.base_testcase import TransactionalTestCase
 
 
 class TestAdapters(TransactionalTestCase):
-    def test_wavelet_adapter(self, tmpdir, session, operation_factory):
+    def test_wavelet_adapter(self, tmpdir, session, operation_factory, time_series_index_factory):
         storage_folder = str(tmpdir)
-        ts_index = make_ts_from_op(session, operation_factory)
+        ts_index = time_series_index_factory()
```

Comment:
Do you still need arguments session and operation_factory in this method or the ones following bellow?